src/hooks/staticContent.ts

```diff
@@ -72,7 +72,7 @@ export const useStaticContent = <T>({
 
     try {
```

Comment:
You said we don't need the try catches any more, but we want to keep it here, for the case the `parseFromJson` function throws an error?

Reply:
true, i also thought about it and decided to leave it as is due to uncertainty on my part in this point